common/src/main/java/com/wynntils/core/notifications/MessageContainer.java

```diff
@@ -29,4 +37,10 @@ public void editMessage(String newMessage) {
     public void update(MessageContainer other) {
```

Comment:
Oh no. It gets worse. :( I'm seriously starting to think we need to redesign this class and its interactions with the manager. I looked up this as well. It, too, is only called once, from the TimedMessageContainer, which in turn is called from the notification overlay. I don't fully understand the code, but it is used to take an existing message and reset its lifespan.